docs/sampler.md

```diff
@@ -115,6 +111,28 @@ optional arguments:
                         Number of sampling trials for each sample size.
 ```
 
+## Reference traces
+
+The reference traces are stored in `data/reference` folder of this repository, as `preprocessed.tar.gz` and
+`sampled.tar.gz` files stored in Git LFS.
+
+`preprocessed.tar.gz` contains the preprocessed traces for the original Azure trace for day 1, 00:09:00-00:11:30 (150
+minutes total).
+
+`sampled.tar.gz` contains the sampled traces for preprocessed trace from `preprocessed.tar.gz`. Sample sizes are 50-3k
+functions with step 50 and 3k-24k with step 1k.
+
+The reference traces were obtained by running the following commands:
+
+```console
+python3 -m preprocess  -t data/azure/ -o data/reference/preprocessed_150 -s 00:00:09 -dur 150
```

Comment:
Inconsistency with the line 119. Did you start preprocessing from the 9th hour, or the 9th second?

Reply:
Yes, I made a mistake there. But the 9am is written as 00:09:00 because the format is dd:hh:mm, not the usual hh:mm:ss.